src/schema-form.js

```diff
@@ -20,6 +21,7 @@ export const SchemaForm = ({
     const [obj, setObj] = useState(Object.assign({}, data))
     const [keys] = useState(Object.keys(schema.properties))
     const [instance] = useState(new UISchema(schema))
+    const [registry] = useState(new ComponentRegistry())
```

Comment:
Expose component registry as singleton for now. The reason behind this is that the user can register components once at application bootstrap and each usage of `SchemaForm` will make use of that already configured registry